templates/clusterctl.yaml

```diff
@@ -57,7 +57,7 @@ ETCD_VERSION: v3.4.13_vmware.14 # Ignore this property if you are using one of t
 DNS_VERSION: v1.7.0_vmware.12 # Ignore this property if you are using one of the existing flavors
 POD_CIDR: "100.96.0.0/11"
 SERVICE_CIDR: "100.64.0.0/13"
-TKR_VERSION: v1.20.8---vmware.1-tkg.2 # Ignore this property if you are using one of the existing flavors
+TARGET_TANZUKUBERNETESRELEASE: ""
```

Comment:
Sorry Richard, you did ask me whether to leave it empty and I said yes.
Can you please give a valid value so that users can see this as an example? See one of our RDEs that UI generates to see the format?